File: Python/Warmup-1/not_string.py
# Given a string, return a new string where "not " has been added to the front.
# However, if the string already begins with "not", return the string unchanged.

def not_string(str):
  if 3 <= len(str) and str[:3] == 'not':
    return str
  return 'not ' + str

File: Java/Warmup-1/notString.java
/*
 * Given a string, return a new string where "not " has been added to the front. However, 
 * if the string already begins with "not", return the string unchanged. 
 * Note: use .equals() to compare 2 strings.
 */

public String notString(String str) {
  if(3 <= str.length() && str.substring(0, 3).equals("not")){
    return str;
  }
  return "not " + str;
}
